### ISSUE TITLE
Question about traffic initiated from the web server

### ISSUE THREAD

**Issue by Vinch157** [PaloAltoNetworks/aws-elb-autoscaling#17]
Hello,

In the routing table of the VPC created by the application template (pan_aws_nlb_vpc-2.0.template) the routing table shows a default route pointing to the IGW of the VPC. It means that traffic initiated from the web server in the VPC will not go through the autoscaling cluster of firewalls deployed by the firewall-v2.0.template. Is this correct? If yes, it means that outgoing traffic is not protected (data exfiltration or connections to known malicious URLs etc)?

Thank you.

**Comment by narayan-iyengar** [comment]
Yes. That is by design. The solutions primary use case is inbound protection. For outbound and inter vpc we recommend using the transit vpc.

Mainly'cauwe scaling requirements for inbound and outbound are different and we don't want to inundate inbounds firewall with outbound or inter vpc traffic.


--
Thanks,
/narayan
________________________________

**Comment by originalwarby** [comment]
Our Transit VPC solution is published here: https://github.com/PaloAltoNetworks/aws-transit-vpc

Light board video if you're interested is her: https://www.paloaltonetworks.com/resources/videos/firewall-services-vpc-integration

HTH

**Comment by Vinch157** [comment]
Thank you this is clear.
I'm indeed trying to "merge" the autoscaling VPC and the transit VPC.
So, if I want to protect the traffic initiated from the web server in the VPC created by the application template (pan_aws_nlb_vpc-2.0.template), it means that I need to change the default route and make it point to a transit VPC (and this VPC pan_aws_nlb_vpc-2.0.template becomes in fact a Subscriber VPC) in place of pointing to the IGW.
But when the default gateway of the route table containing the web server does not point to the IGW (and point to the transit VPC), traffic initiated from outside to reach the web server via autoscaling firewalls is blackholed. How can I avoid this?

**Comment by originalwarby** [comment]
It won't be black-holed.  Have a look at my response to you in the transit-vpc repo.
